### PR TITLE
home page

### PR DESCRIPTION
**Pull request by derwehr**
implements #312

### CODE REVIEW

File: frontend/src/home/HomeVideoModal.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+const useStyles = makeStyles(() =>
+  createStyles({
+    button: {
+      color: '#fff',
+      backgroundColor: deepPurple[500],
+      '&:hover': {
+        backgroundColor: purple[500],
+      },
+    },
+    iframe: {
```
Comment (PianoRollRepresentation) [inline frontend/src/home/HomeVideoModal.tsx:15]: this scss class is not used. And dont use fixed pixels

File: frontend/src/home/HomeVideoModal.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const body = (
+    <Dialog maxWidth="xl" open={open} onClose={handleClose}>
+      {/* youtube iFrame */}
+      <iframe
+        className={classes.iframe}
+        width="1280"
```
Comment (PianoRollRepresentation) [inline frontend/src/home/HomeVideoModal.tsx:39]: use width and height from scss or inline but not both. Dont use fixed pixels

File: frontend/src/home/HomeVideoModal.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    </Dialog>
+  );
+
+  return (
+    <>
+      <Button
+        variant="contained"
+        className={classes.button}
+        onClick={handleOpen}
+        style={{ marginBottom: '45px' }}
```
Comment (PianoRollRepresentation) [inline frontend/src/home/HomeVideoModal.tsx:56]: dont use fixed pixels. Use '5vh' here

**Review by PianoRollRepresentation — CHANGES_REQUESTED** [review]

File: frontend/src/home/HomeVideoModal.tsx (hunk 1 above)
Comment (derwehr) [reply frontend/src/home/HomeVideoModal.tsx:15]: done

File: frontend/src/home/HomeVideoModal.tsx (hunk 2 above)
Comment (derwehr) [reply frontend/src/home/HomeVideoModal.tsx]: done

File: frontend/src/home/HomeVideoModal.tsx (hunk 3 above)
Comment (derwehr) [reply frontend/src/home/HomeVideoModal.tsx:54]: This margin is used to keep the button and arrow image next to it alligned.
Using 5vh here won't do that on smaller displays. I believe in this case using fixed pixels is the best way, as button and image will both always have the same height.

**Review by PianoRollRepresentation — APPROVED** [review]

**Pull request merged**